tests/confs/bad-autopush_verified_push.conf

```diff
@@ -0,0 +1,7 @@
+[duo]
+ikey = DIXYZV6YM8IFYVWBINCA
+skey = yWHSMhWucAcp7qvuH3HWTaSaKABs8Gaddiv1NIRo
+host = localhost:4443
+cafile = certs/mockduo-ca.pem
```

Comment:
Not sure if this is a big deal, since it shouldn't be used, but there's no cafile parameter in the corresponding Python object above

Reply:
Sharp eye! `cafile` isn’t needed, so I removed it